app/timetables/admin.py

```diff
@@ -97,7 +97,7 @@ class EventAdmin(TimeStampAdmin):
 class VendorAdmin(DefaultAdmin):
     """Admin customisation for Vendor model."""
 
-    fields = ('name', 'slug', 'info', 'start_date', 'end_date')
+    fields = ('name', 'slug', 'info')
```

Comment:
@andela-kerinoso why are we removing the start date and end date from the vendor?